Sources/Core/Spec+Context.swift

```diff
@@ -0,0 +1,51 @@
+//  Spec+Context.swift
+
+import Foundation
+
+extension Spec.Product {
+    func makeContext() -> [String: Any] {
+        [
+            "name": name,
+            "productType": productType.rawValue,
```

Comment:
Although the keys in the rest of the file use snake casing, I left this one to match the name of the property to avoid breaking backward compatibility with the Stencil file provided as part of the current major version.